README.md

```diff
@@ -1,18 +1,25 @@
-# harmony-gdal
+# harmony-gdal-adapter
 
 ![](https://data-services-github-badges.s3.amazonaws.com/cov.svg?dummy=true)
 
-<img src="https://data-services-github-badges.s3.amazonaws.com/cov.svg?dummy=hello" />
+A demonstration of a subsetter capability to be used with Harmomy.
```

Comment:
I think at this point we're probably beyond HGA being a demonstration of a subsetter. Maybe "A [Harmony](https://harmony.earthdata.nasa.gov/) backend service that transforms input images using GDAL." (Or something similar)

Reply:
(I forgot that GitHub would automatically render my Markdown - but I'd intended for that Harmony URL to be in the README, not just a link in this comment)